src/NuGet.Core/NuGet.Versioning/VersionRangeFactory.cs

```diff
@@ -204,7 +204,7 @@ public static bool TryParse(string value, bool allowFloating, [NotNullWhen(true)
 #if NETCOREAPP2_1_OR_GREATER
                 if (allowFloating && minVersionString.Contains('*', StringComparison.Ordinal))
```

Comment:
Does this have the same problem?

Reply:
Additional overloads for `string.Contains` were added in 2.1+, so this call doesn't have the same issue.